content/tutorials/vignette_n2khab_v025_geospatial_hab/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Working with geospatial data sources of habitat (sub)types and RIBs"
+description: "Vignette for the R package n2khab."
+authors: [florisvdh, Toon Westra, w_jan, cecileherr]
```

Comment:
You need to make sure that for every author there’s a `AUTHOR_ID.toml` file under `data/authors` and the match with that ID must be exact; see https://inbo.github.io/tutorials/create_tutorial/. The above is just an example; there are several cases where this is still a problem. Once a commit is added that solves this, the GH action workflow that builds the website will hopefully succeed so that you can preview the result. Currently this failed.

Reply:
I will fix this

Reply:
Is this case sensitive?

Reply:
Apparently it is. And horrible to debug. 🤪